packages/flutter/lib/src/material/list.dart

```diff
@@ -44,7 +45,15 @@ class MaterialList extends StatefulWidget {
 }
 
 class _MaterialListState extends State<MaterialList> {
-  ScrollbarPainter _scrollbarPainter = new ScrollbarPainter();
+  ScrollbarPainter _scrollbarPainter;
+
+  @override
+  void initState() {
+    super.initState();
+    _scrollbarPainter = new ScrollbarPainter(
+      getThumbColor: () => Theme.of(context).highlightColor
```

Comment:
It's slightly more efficient to use a member function than a closure here.  The issue is that the closure captures the local variables in this scope whereas a member function only captures `this`, which is specially optimized in the VM.